WowPacketParser/SQL/SQLDatabase.cs

```diff
@@ -94,54 +94,74 @@ private static void LoadBroadcastText()
             string query =
                 "SELECT ID, Text, Text1, EmoteID1, EmoteID2, EmoteID3, EmoteDelay1, EmoteDelay2, EmoteDelay3, EmotesID, LanguageID, Flags, ConditionID, SoundEntriesID1, SoundEntriesID2 " +
                 $"FROM {Settings.HotfixesDatabase}.broadcast_text;";
-            using (var reader = SQLConnector.ExecuteQuery(query))
+
+            if (Settings.TargetedDatabase < TargetedDatabase.Cataclysm)
```

Comment:
3.3.5 branch also has broadcast_text table in world db

Reply:
solved